### PR TITLE
Inject version info into index.html

### PR DESCRIPTION
**Pull request by yurishkuro**
Part of https://github.com/jaegertracing/jaeger-ui/issues/365

### CODE REVIEW

File: pkg/version/build.go [hunk 1]
```diff
@@ -28,8 +28,8 @@ var (
 // Info holds build information
 type Info struct {
 	GitCommit  string `json:"gitCommit"`
-	GitVersion string `json:"GitVersion"`
-	BuildDate  string `json:"BuildDate"`
+	GitVersion string `json:"gitVersion"`
+	BuildDate  string `json:"buildDate"`
 }
```
Comment (yurishkuro) [inline pkg/version/build.go:33]: This could be considered a minor breaking change, I don't know why there was a mismatch in the name casing.

Comment (yurishkuro) [reply pkg/version/build.go:33]: This JSON output is used in `version` command and `/version` endpoint:
```
$ curl http://localhost:14269/version
{"gitCommit":"","gitVersion":"","buildDate":""}
```
Hard to imaging there would be dependencies on the exact string output of these.

File: cmd/query/app/static_handler.go [hunk 2]
```diff
@@ -113,14 +116,19 @@ func loadIndexBytes(open func(string) (http.File, error), options StaticAssetsHa
...
+	versionString := fmt.Sprintf("JAEGER_VERSION = %s;", string(versionJSON))
+	indexBytes = versionPattern.ReplaceAll(indexBytes, []byte(versionString))
+	// replace base path
 	if options.BasePath == "" {
 		options.BasePath = "/"
 	}
 	if options.BasePath != "/" {
 		if !strings.HasPrefix(options.BasePath, "/") || strings.HasSuffix(options.BasePath, "/") {
-			return nil, fmt.Errorf(errBadBasePath, options.BasePath)
+			return nil, fmt.Errorf("invalid base path '%s'. Must start but not end with a slash '/', e.g. '/jaeger/ui'", options.BasePath)
```
Comment (jpkrohling) [inline cmd/query/app/static_handler.go:129]: How about replacing `'%s'` with `%q` ?

File: pkg/version/build.go (hunk 1 above)
Comment (jpkrohling) [reply pkg/version/build.go:33]: I really wouldn't worry about it...

**Review by jpkrohling — APPROVED** [review]

**Review by joe-elliott — APPROVED** [review]
Looks good to me.

File: cmd/query/app/static_handler.go (hunk 2 above)
Comment (yurishkuro) [reply cmd/query/app/static_handler.go:129]: I think it only makes the logs uglier because JSON logger will have to escape quotes, etc.

**Pull request merged**